### DISCUSSION TITLE
Lesson 6: Brownie Fund Me - Unable to expand environment variable in host setting

### DISCUSSION THREAD

**Discussion by realglass** [smartcontractkit/full-blockchain-solidity-course-py#1559] [Q&A] [answered]
Hi!! 

At around 5:15 I am encountering this error. Any help, please? Error: Unable to expand environment variable in host setting

![image](https://user-images.githubusercontent.com/101928609/176440084-48bdaa4c-de21-46dd-921c-f0365493061a.png)

**Comment by cromewar** [comment, marked as answer]
Hello @realglass 
Usually the export keyword is used for Unix based systems, linux and Mac... so please delete de export keyword, and also delete the spaces between `=`.
Finally as you are on Windows is preferable to add those variables trough windows advanced configuration into Environment Variables.
Cheers.

**Reply by realglass** [reply]
Hi @cromewar 
Thank you for your reply! Truly appreciated.  Unfortunately, the problems still persist. After removing the export most of the errors stopped but 1 still persists. Also when I remove the spaces between the = and save they reappear again. Also, note that the environmental variables were set using the advanced configuration. Any idea? 

![image](https://user-images.githubusercontent.com/101928609/176853655-9f52a249-f0bf-45ba-90bb-b5a1dd30f001.png)
![image](https://user-images.githubusercontent.com/101928609/176853862-6da4629e-3225-4a57-a292-8b6b8c17347c.png)

**Reply by cromewar** [reply]
2 things:
Install the env extension on the vscode extension store, 
and also remember to save the brownie-config.yaml and add the `dotenv` identifier.

Does it works even if the editor complains?

**Reply by realglass** [reply]
Thank you so much sir!!! <3 
problem fixed. did not have env extension installed

**Reply by cromewar** [reply]
I'm glad is solved now.
